.github/workflows/_reusable-eng-tools-test.yaml

```diff
@@ -8,6 +8,14 @@ on:
         description: Name of package under eng/tools
         required: true
         type: string
+      lint:
```

Comment:
Fine for now, but after this PR, I think we should enable lint and prettier consistently for all packages under eng.